scripts/common_startup.sh

```diff
@@ -18,6 +18,7 @@ CREATE_VENV=1
 REPLACE_PIP=$SET_VENV
 COPY_SAMPLE_FILES=1
 SKIP_CLIENT_BUILD=${GALAXY_SKIP_CLIENT_BUILD:-0}
+NODE_VERSION="10.13.0"
```

Comment:
As soon as I merged it, I realized I think this should be more readily configurable. What do you think about?

```
NODE_VERSION=${GALAXY_NODE_VERSION:-"10.13.0"}
```

Not a big deal either way though.

Reply:
Hrmm, maybe?  Can't hurt, but I don't know that we really want people configuring this one.  If they want to run their own node/yarn and try different client builds, they already can.

The more I think about this, I want to backport it to 18.09 and merge forward with pinned node versions starting there, so I can add it when I do that if you want.